src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -3801,16 +3801,6 @@ public class EventListener : IDisposable
         /// </summary>
         public event EventHandler<EventWrittenEventArgs>? EventWritten;
 
-        static EventListener()
-        {
-#if FEATURE_PERFTRACING
```

Comment:
Is `NativeRuntimeEventSource` now guaranteed to always be initialized, even just for a console app?